src/Compilers/Core/Portable/Emit/EditAndContinue/EmitBaseline.cs

```diff
@@ -70,16 +70,16 @@ internal sealed class MetadataSymbols
             public readonly IReadOnlyDictionary<AnonymousTypeKey, AnonymousTypeValue> AnonymousTypes;
 
             /// <summary>
-            /// In C#, the set of anonymous delegates where the parameter types and return type are
-            /// generic type arguments; in VB, this set is unused and empty.
+            /// In C#, the set of anonymous delegates with their type fully serialized into their name;
```

Comment:
>with their type fully serialized into their name [](http://example.com/codeflow?start=54&length=48)

I am not sure how to interpret this phrase. I read it as: "a type of an anonymous delegate is fully serialized into its name." What is not clear to me is what is the "type of an anonymous delegate" in this context.

Reply:
Thanks, clarified.